### PR TITLE
docs(subplat): add initial product config documentation

### PR DESCRIPTION
**Pull request by chenba**

### CODE REVIEW

**Comment by chenba** [comment]
@chelsea-lewis Anna's the collaborator on the jira for this PR but obviously she cannot review it now.  Let me know who should review this.  Thanks!

**Comment by chelsea-lewis** [comment]
Thanks for the flag @chenba - I've added Reino as primary reviewer, and I'll plan to review as well. Updated https://mozilla-hub.atlassian.net/browse/FXA-3959 in Jira.

File: docs/tutorials/subscription-platform.md [hunk 1]
```diff
@@ -109,14 +109,115 @@ Some of the metadata properties listed above [have defaults][product-details-def
...
+:::
+
+###### Top Level Configuration Document Values
+
+The configuration document is a JSON object.  The top level keys for that object are below.
+
+| Key                | Description |
+| ------------------ | ----------- |
+| id                 | An identifier for the system storing the document.  It's not intended for manual edits.|
+| productConfigId    | _Plan configuration only_.  The identifier of the product configuration document from which the plan configuration inherits its default values. |
```
Comment (StaberindeZA) [inline docs/tutorials/subscription-platform.md:152]: Some parameters are listed as `Optional` others as `Required` and some have none. The one's with none might be a little confusing whether or not its `Optional` or `Required`.

**Review by StaberindeZA — APPROVED** [review]
LGTM. It's nice having all the links and the example JSON files.

File: docs/tutorials/subscription-platform.md (hunk 1 above)
Comment (chenba) [reply docs/tutorials/subscription-platform.md:152]: I'll remove the `Required` and mentioned that they are required unless otherwise noted.

**Pull request merged**